### PR TITLE
DEV-3890-Test & Solution argsort

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: solutions/argsort/main.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+				if argument[0][i] > argument[0][j] {
+					z01.PrintRune('F')
+					z01.PrintRune('\n')
+					return
+				}
+			}
+		}
+		z01.PrintRune('T')
+	}
+	z01.PrintRune('\n')
```
Comment (Hamzaelkhatri) [inline solutions/argsort/main.go:24]: in the subject, you say that the program should print nothing if   `the number of arguments is not `1` Print nothing`. but here the program can print `\n` in any case

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: solutions/argsort/main.go (hunk 1 above)
Comment (zainabdnaya) [reply solutions/argsort/main.go:24]: I changed the subject

File: tests/argsort_test/main.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"github.com/01-edu/go-tests/lib/challenge"
+)
+
+func main() {
+	args := [][]string{
+		{"abc"},
+		{"abc", "def"},
+		{"abc", "def", "ghi"},
+		{"125", "wde"},
+		{"125", "wde", "abc"},
```
Comment (Hamzaelkhatri) [inline tests/argsort_test/main.go:13]: add empty string 
```
{""}
{}

**Review by Hamzaelkhatri — APPROVED** [review]

**Review by Hamzaelkhatri — APPROVED** [review]

File: tests/argsort_test/main.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	"github.com/01-edu/go-tests/lib/challenge"
+)
+
+func main() {
+	args := [][]string{
+		{"abc"},
+		{"abc", "def"},
+		{"125"},
+		{"125", "wde"},
+		{"def"},
```
Comment (MSilva95) [inline tests/argsort_test/main.go:13]: add more tests, with other printable characters, and with spaces at the end and middle of the string

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/argsort_test/main.go (hunk 3 above)
Comment (zainabdnaya) [reply tests/argsort_test/main.go:13]: Done

File: tests/argsort_test/main.go [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+		{""},
+		{"123qa"},
+		{"	 HELLO!"},
+		{" kowz"},
+		{"a b c"},
+		{"   abc"},
+		{"    hello"},
+		{"hello    "},
+		{"sto  ry"},
+		{"  @\!\\n"},
```
Comment (MSilva95) [inline tests/argsort_test/main.go:23]: did you try to run the test locally?
Was it possible? i cant run it with this slashes...

File: tests/argsort_test/main.go [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+		{"9^_\`abc"},
```
Comment (MSilva95) [inline tests/argsort_test/main.go:24]: same here...

**Review by MSilva95 — DISMISSED** [review]

**Review by MSilva95 — COMMENTED** [review]
My comments have not been answered and no changes have been made, why is this here to review again?

**Comment by zainabdnaya** [comment]
> My comments have not been answered and no changes have been made, why is this here to review again?

I did but I didn't submit it  
it fixed now

File: tests/argsort_test/main.go (hunk 5 above)
Comment (MSilva95) [reply tests/argsort_test/main.go:24]: if i try to run it like it is it gives me an error 
test
Yours works fine?

Comment (MSilva95) [reply tests/argsort_test/main.go:24]: mine only works if i set the tests like so :
```
{"  @\\!\\n"},
{"9^_\\`abc"},
```

Comment (zainabdnaya) [reply tests/argsort_test/main.go:24]: You're right. I had a problem with commits; it didn't show this problem, but after I fixed it, it showed the same msg.

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

**Review by tiagocollot — APPROVED** [review]
LGTM! ⚡

**Pull request merged**